packages/reusable-transforms/src/rename-jsx-component/verifyIsJsxIdentifierOrAddComment.ts

```diff
@@ -0,0 +1,29 @@
+import { ASTPath, JSCodeshift } from "jscodeshift";
+import { namedTypes } from "ast-types";
+import { joinWith, tick } from "nice-comment";
+
+import { insertMultilineComment } from "@codeshift/utils";
+
+export const verifyIsJsxIdentifierOrAddComment = (
+    j: JSCodeshift, //
+    path: ASTPath<any>,
+    name: string,
+    node: any,
+): node is namedTypes.JSXIdentifier => {
+    if (j.JSXIdentifier.check(node)) {
+        return true;
+    }
+
+    const comment = joinWith(" ")([
+        `ERROR expected`, //
+        name,
+        `to be`,
+        tick(j.jsxIdentifier("").type, ","), //
+        `got`,
+        tick(node.type, "."), //
+    ]);
```

Comment:
I know we discussed this already, but I'll note it here as well that to me, a template literal would be more readable to me, compared to this array concatenation.

```suggestion
    const comment = `ERROR expected ${name} to be \`${j.jsxIdentifier("").type}\`, got \`${node.type}\`.`;
```

Reply:
done.